src/nodedef.cpp

```diff
@@ -1456,7 +1454,7 @@ void NodeDefManager::updateTextures(IGameDef *gamedef,
 	for (u32 i = 0; i < size; i++) {
 		ContentFeatures *f = &(m_content_features[i]);
 		f->updateTextures(tsrc, shdsrc, meshmanip, client, tsettings);
-		progress_callback(progress_callback_args, i, size);
+		((Client *)gamedef)->showUpdateProgressTexture(progress_callback_args, i, size);
```

Comment:
`client` exists in this function with the correct type

Reply:
fixed, my bad :)